README.md

```diff
@@ -23,13 +23,43 @@ Documentation can be generated with [ExDoc](https://github.com/elixir-lang/ex_do
 and published on [HexDocs](https://hexdocs.pm). Once published, the docs can
 be found at <https://hexdocs.pm/tigerbeetlex>.
 
+## Contributing
+
+Prerequisites: The current beta version of TigerBeetle is pinned to zig `0.91`, you'll need a zig executable binary in your `PATH` to compile the tigerbeetle git tree dependency in `./src/tigerbeetle` and a running tigerbeetle instance running on `127.0.0.1:3000`:
+
+### Setup
+
+```bash
+$ git clone https://github.com/rbino/tigerbeetlex.git
+$ cd tigerbeetlex/src/tigerbeetle
+$ git submodule update --init
+$ scripts/install.sh
+$ ./tigerbeetle format --cluster=0 --replica=0 --replica-count=1 0_0.tigerbeetle
+$ ./tigerbeetle start --addresses=3000 0_0.tigerbeetle
+```
+
+see the [tigerbeetle documentation](https://docs.tigerbeetle.com/) for more info.
+In a new terminal session:
+
+```bash
+$ cd tigerbeetlex
+$ mix deps.get
+$ mix deps.compile
```

Comment:
No need to compile the deps here (the command compiles deps for the default `dev` environment, and then when you run `mix test` it recompiles them for the `test` environment)